src/stylesheets/components/_navigation.scss

```diff
@@ -1,41 +1,66 @@
 .app-navigation {
+  border-bottom: 1px solid $govuk-border-colour;
   background-color: $app-light-grey;
 
-  @include govuk-media-query($until: tablet) {
+  @include govuk-media-query($from: tablet) {
+    display: block;
+  }
+
+  [hidden],
+  [hidden="hidden"],
```

Comment:
Ooh, I've not seen this before. I think in govuk-frontend, we've just used `[hidden]` previously. Is it needed for certain browsers?

Reply:
I was finding when I tested this that I couldn't get the hidden rules to override as in some cases the tag I was trying to hide had `hidden="hidden"` instead of just `hidden` so it would ignore the `hidden` rule. I'm now wondering if I could just set the js to have a blank `setAttribute`, so `setAttribute('hidden', '')` instead of `setAttribute`, so `setAttribute('hidden', 'hidden')`. Lemme have a look at this.